index.js

```diff
@@ -30,16 +38,31 @@ store.subscribe(() => {
 })
 
 store.dispatch({ type: 'ADD_BOOK', title: 'Война и мир, Voïna i mir' })
-assert(Immutable.is(Map({books: Set().add('Война и мир, Voïna i mir'), customers: Set()}), store.getState()) === true)
+let map = Map()
+  .set('books', Set().add('Война и мир, Voïna i mir'))
+  .set('customers', Set())
+assert.deepStrictEqual(map.toJS(), store.getState().toJS())
```

Comment:
No need to create an temporary immutable value, and convert it to JS plain values right away.